tests/vlan_test.py

```diff
@@ -0,0 +1,455 @@
+
+    def test_config_vlan_add_exist_port_member(self):
+        runner = CliRunner()
+        result = runner.invoke(config.config.commands["vlan"].commands["member"].commands["add"], ["1000", "Ethernet4"])
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code != 0
+        assert "Error: Ethernet4 is already a member of Vlan1000" in result.output
+
+    def test_config_vlan_add_nonexist_port_member(self):
+        runner = CliRunner()
+        result = runner.invoke(config.config.commands["vlan"].commands["member"].commands["add"], ["1000", "Ethernet3"])
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code != 0
+        assert "Error: Ethernet3 does not exist" in result.output
+
+    def test_config_vlan_add_nonexist_portchannel_member(self):
+        runner = CliRunner()
+        result = runner.invoke(config.config.commands["vlan"].commands["member"].commands["add"], \
+				["1000", "PortChannel1011"])
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code != 0
+        assert "Error: PortChannel1011 does not exist" in result.output
+
+    def test_config_vlan_add_portchannel_member(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(config.config.commands["vlan"].commands["member"].commands["add"], \
+				["1000", "PortChannel1001", "--untagged"], obj=db)
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code == 0
+
+        # show output
+        result = runner.invoke(show.cli.commands["vlan"].commands["brief"], [], obj=db)
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code == 0
+        assert result.output == show_vlan_brief_with_portchannel_output
+
+    def test_config_vlan_add_rif_portchannel_member(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(config.config.commands["vlan"].commands["member"].commands["add"], \
+				["1000", "PortChannel0001", "--untagged"], obj=db)
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code != 0
+        assert "Error: PortChannel0001 is a router interface!" in result.output
+
+    def test_config_vlan_del_vlan(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(config.config.commands["vlan"].commands["del"], ["1000"], obj=db)
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code == 0
+
+        # show output
+        result = runner.invoke(show.cli.commands["vlan"].commands["brief"], [], obj=db)
+        print(result.exit_code)
+        print(result.output)
+        assert result.exit_code == 0
+        assert result.output == show_vlan_brief_empty_output
+
+    def test_config_vlan_del_nonexist_vlan_member(self):
+        runner = CliRunner()
+        db = Db()
```

Comment:
LGTM flagged this as unused. Did you mean to pass to `runner.invoke()`?

Reply:
loos like lgtm does not warning it. fixed.